pytest_fixtures/broker.py

```diff
@@ -67,10 +67,19 @@ def rhel7_contenthost_module():
 @pytest.fixture
 def rhel8_contenthost():
     """A fixture that provides a content host object based on the rhel8 nick"""
+    pytest.skip('Skipped until rhel8 templates have wget added')
```

Comment:
Why is `wget` necessary to use the fixture? You can provision and register a system without it.

Reply:
@JacobCallahan is this because of `wget` in `robottelo/hosts.py `? If so, see:
https://github.com/SatelliteQE/robottelo/pull/8556#discussion_r625625501

Reply:
@swadeley iirc there may be other places where it is needed, but after that change is introduced I can rebase and test again.

Reply:
yep. just confirmed that wget is used by helper methods elsewhere. i'm fine with us waiting on a rhel8 template that meets our requirements.